lib/ansible/module_utils/vmware.py

```diff
@@ -27,19 +28,7 @@ class TaskError(Exception):
 
 
 def wait_for_task(task):
```

Comment:
you can do 
```python
wait_for_task = PyVmomi.wait_for_task
```
or vice versa, but I don't see a point of such duplication aliasing.

Why did you move it to class method in the first place?

Reply:
We are moving every function inside `PyVmomi` class, but we are doing this gradually so once all modules migrate to `PyVmomi` class, we will remove this function all together.

Reply:
I would suggest changing calling this in this module to calling the static method instead. But on the other side, is it really needed? What's wrong with having utility functions separate from class? They are not related to instances, so it's logical to keep them separate.